test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/ReadWrite/Updating/Relationships/AddToToManyRelationshipTests.cs

```diff
@@ -0,0 +1,107 @@
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using JsonApiDotNetCore.Serialization.Objects;
+using Xunit;
+
+namespace JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.ReadWrite.Updating.Relationships
+{
+    public sealed class AddToToManyRelationshipTests
```

Comment:
As mentioned earlier, I'd prefer to create the related resources in the database upfront for these tests, because they are being referenced in the request.